tests/automation/call_checks.spec.ts

```diff
@@ -1,32 +1,26 @@
 import { sleepFor } from '../promise_utils';
-import { newUser } from './setup/new_user';
+import { test_Alice_1W_Bob_1W } from './setup/sessionTest';
 import { createContact } from './utilities/create_contact';
 import { clickOnMatchingText, clickOnTestIdWithText } from './utilities/utils';
-import { sessionTestTwoWindows } from './setup/sessionTest';
 
-sessionTestTwoWindows('Voice calls', async ([windowA, windowB]) => {
-  const [userA, userB] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-  ]);
-
-  await createContact(windowA, windowB, userA, userB);
-  await clickOnTestIdWithText(windowA, 'call-button');
-  await clickOnTestIdWithText(windowA, 'session-toast');
-  await clickOnTestIdWithText(windowA, 'enable-calls');
-  await clickOnTestIdWithText(windowA, 'session-confirm-ok-button');
-  await clickOnTestIdWithText(windowA, 'message-section');
+test_Alice_1W_Bob_1W('Voice calls', async ({ alice, alice1, bob, bob1 }) => {
+  await createContact(alice1, bob1, alice, bob);
+  await clickOnTestIdWithText(alice1, 'call-button');
+  await clickOnTestIdWithText(alice1, 'session-toast');
+  await clickOnTestIdWithText(alice1, 'enable-calls');
+  await clickOnTestIdWithText(alice1, 'session-confirm-ok-button');
+  await clickOnTestIdWithText(alice1, 'message-section');
   await clickOnTestIdWithText(
-    windowA,
+    alice1,
```

Comment:
Should this also have alice1W?

Reply:
or actually aliceW1 for when we're using the specific window?

Reply:
just to differentiate between when we're using the window alice and not the username alice...?